services/traction/api/tasks/present_proof_tasks.py

```diff
@@ -0,0 +1,168 @@
+import logging
+from re import Pattern, compile
+from uuid import UUID
+from acapy_client.model.indy_proof_req_attr_spec import IndyProofReqAttrSpec
+from acapy_client.model.indy_proof_req_pred_spec import IndyProofReqPredSpec
+from acapy_client.model.indy_proof_request import IndyProofRequest
+
+from acapy_client.model.v10_presentation_send_request_request import (
+    V10PresentationSendRequestRequest,
+)
+
+
+from sqlalchemy import update
+from sqlalchemy.exc import DBAPIError
+
+
+from api.tasks.tasks import Task, TractionTaskType
+from api.db.models import Tenant
+from api.db.models.v1.contact import Contact
+from api.endpoints.models.credentials import ProofRequest
+from api.db.session import async_session
+
+from acapy_client.api.present_proof_v1_0_api import PresentProofV10Api
+from api.api_client_utils import get_api_client
+
+from api.db.models.v1.presentation_requests import VerifierPresentationRequest
+
+TRACTION_TASK_PREFIX = "traction::TASK::"
```

Comment:
let's start a parent file or something that has this in it and the abstract class. i think we should follow your lead here and get the other tasks into separate files (backlog)

Reply:
keeping the constants in the main file, to be refactored later.